prereise/gather/data/remap_ba_area/remap_ba_area.py

```diff
@@ -0,0 +1,294 @@
+import json
+import os
+import shutil
+
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import pandas as pd
+import requests
+from pyproj import CRS
+
+from .map_data import ca_name_match, ca_name_missing, states_to_map_with_counties
```

Comment:
When I try running the script from the command line, I get an error:
```
Traceback (most recent call last):
  File "C:\Users\DanielOlsen\repos\bes\PreREISE\prereise\gather\data\remap_ba_area\remap_ba_area.py", line 11, in <module>
    from .map_data import ca_name_match, ca_name_missing, states_to_map_with_counties
ImportError: attempted relative import with no known parent package
```

Reply:
It's the relative path issue. Let me change to the absolute import.